__tests__/api.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require('../app.js');
+const request = require('supertest');
+
+const seed = require('../db/seeds/seed.js')
+const testData = require('../db/data/test-data')
+const connection = require('../db/connection.js')
+
+beforeEach(() => seed(testData))
+afterAll(() => connection.end());
+
+describe("GET /api/topics", () => {
+  describe("successful connection test(s)", () => {
+    test("200: expect page to respond with corresponding topics table", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({
+          body
+        }) => {
+          expect(body).toMatchObject(testData.topicData)
+          expect(body).toHaveLength(3)
+          for (let i = 0; i < body.length; i++) {
+            expect(body[0]).toHaveProperty("slug", expect.any(String))
+            expect(body[0]).toHaveProperty("description", expect.any(String))
+          }
```

Comment:
As you are only checking the first index in your expect statements is this loop doing anything? 👀 . Now you have a length check would a forEach maybe more appropriate here? You could then check all the indexes have the properties you expect. 

Nice work of the use expect.any() for your values making them nice and dynamic 🤸